### DISCUSSION TITLE
[react-hooks] useQueries support

### DISCUSSION THREAD

**Discussion by michael-wolfenden** [ecyrbe/zodios#345] [Q&A] [answered]
I'm would like to use react query's suspense support with `useQueries`  to prevent [suspense waterfalls when I have multiple queries](https://tkdodo.eu/blog/seeding-the-query-cache#prefetching), but I couldn't find a way to use `useQueries` with the hooks generated by zodios.

Is this feature supported?

**Comment by ecyrbe** [comment, marked as answer]
No, not a feature supported out of the box, but you can use directly useQueries and use :
- zodios client `get` or `get<alias>` to provide the queryFn,
- zodios hooks client `getKeyByPath` or `getKeyByAlias` to provide the queryKey.
